docs/releases/1.9.txt

```diff
@@ -25,7 +25,18 @@ Python 3.2 and added support for Python 3.5.
 What's new in Django 1.9
 ========================
 
-...
+Ability to perform actions after a transaction commit
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A new hook, :func:`~django.db.transaction.on_commit` has been introduced to make
+it possible to perform actions after a database transaction is successfully
+committed. This is useful when sending emails, creating Celery tasks or invalidate
```

Comment:
"This is useful for tasks such as sending notification emails, creating queued tasks, or invalidating caches."

Current sentence has a grammatical error with "invalidate", and "useful when" suggests that this is an exclusive list. Also I'm not sure there's any reason to specifically name Celery here (I know, I did in the django-transaction-hooks docs), there are other task queues that people use.